src/main/com/yetanalytics/lrs/pedestal/routes.cljc

```diff
@@ -85,22 +85,31 @@
     :enter (fn health-fn [ctx]
              (assoc ctx :response {:status 200 :body ""}))}))
 
-(defn build [{:keys [lrs path-prefix]
-                     :or {path-prefix "/xapi"}}]
+(defn build [{:keys [lrs
+                     path-prefix
+                     wrap-interceptors]
+              :or {path-prefix "/xapi"
+                   wrap-interceptors []}}]
```

Comment:
Now that we have a bunch of arg map keys, you might want to put a docstring explaining them (particularly `:wrap-interceptors`)

Reply:
https://github.com/yetanalytics/lrs/pull/71/commits/533a7b411949b0d043d7b6cfad879d21d9dc753e